classes/class-twenty-twenty-one-customize.php

```diff
@@ -81,6 +81,38 @@ public function register( $wp_customize ) {
 				)
 			);
 
+			/**
+			 * Add excerpt or full text selector to customizer
+			 */
+			$wp_customize->add_section(
+				'theme_settings',
+				array(
+					'title'    => esc_html__( 'Theme settings', 'twentytwentyone' ),
+					'priority' => 30,
+				)
+			);
+
+			$wp_customize->add_setting(
+				'display_excerpt_or_fullpost',
+				array(
+					'capability' => 'edit_theme_options',
+					'default'    => 'excerpt',
+				)
+			);
+
+			$wp_customize->add_control(
+				'display_excerpt_or_fullpost',
+				array(
+					'type'    => 'radio',
+					'section' => 'theme_settings',
+					'label'   => esc_html__( 'On the Posts page, post show:', 'twentytwentyone' ),
+					'choices' => array(
+						'excerpt' => 'Excerpt',
```

Comment:
The text for the choices needs to be translation ready and escaped.

Reply:
Good point, thank you @carolinan for catching that! :+1: 
Note: should use `esc_html__`